docs/content/dagster-cloud/deployment/serverless.mdx

```diff
@@ -37,14 +37,47 @@ Enterprise customers may request a quota increase by [contacting Sales](mailto:s
 
 ## Getting started with Serverless
 
-When you create a new Dagster Cloud organization, you'll need to activate a deployment type: Hybrid or Serverless. Once activated, you'll be prompted to fork the starter GitHub repo and install the GitHub app. Pushing to the main branch will deploy to your prod Serverless deployment. Pull requests will spin up ephemeral [branch deployments](/dagster-cloud/developing-testing/branch-deployments) using the Serverless agent.
+### With GitHub
 
-If you’re using another git hosting provider, CI service, or simply want to do local development without using a CI service, you can use Serverless via the [dagster-cloud CLI](/dagster-cloud/developing-testing/dagster-cloud-cli). Run the following to learn more:
+If you are a GitHub user, our GitHub integration is the fastest way to get started. It uses a GitHub app and GitHub Actions to set up a repo containing skeleton code and configuration consistent with Dagster Cloud's best practices with a single click.
 
+When you create a new Dagster Cloud organization, you'll be prompted to choose Serverless or Hybrid. Once activated, our GitHub integration will scaffold a new git repo for you with serverless and branch deployments preconfigured. Pushing to the `main` branch will deploy to your `prod` Serverless deployment. Pull requests will spin up ephemeral [branch deployments](/dagster-cloud/developing-testing/branch-deployments) using the Serverless agent.
```

Comment:
When referring to proprietary features, we're considering them to be proper nouns and capitalizing them - so `Serverless` and `Branch Deployments`

When referring to them in a generic context - ex: `a branch deployment`, we should use lower case

```suggestion
When you create a new Dagster Cloud organization, you'll be prompted to choose Serverless or Hybrid deployment. Once activated, our GitHub integration will scaffold a new git repo for you with Serverless and Branch Deployments already configured. Pushing to the `main` branch will deploy to your `prod` Serverless deployment. Pull requests will spin up ephemeral [branch deployments](/dagster-cloud/developing-testing/branch-deployments) using the Serverless agent.
```

Reply:
Additionally, is this correct? I thought this was true if the user opts to clone the starter repo, not if they use an existing one.

> Once activated, our GitHub integration will scaffold a new git repo for you